cmd/test-cli/main.go

```diff
@@ -114,19 +119,19 @@ func doGetPayload(v validatorPrivateData, boostEndpoint string, beaconNode Beaco
 		log.WithError(err).Fatal("could not sign blinded beacon block")
 	}
 
-	payload := boostTypes.SignedBlindedBeaconBlock{
+	payload := consensusapi.SignedBlindedBeaconBlock{
 		Message:   &blindedBeaconBlock,
-		Signature: signature,
+		Signature: phase0.BLSSignature(signature),
 	}
-	var respPayload boostTypes.GetPayloadResponse
+	var respPayload api.VersionedExecutionPayload
 	if _, err := server.SendHTTPRequest(context.TODO(), *http.DefaultClient, http.MethodPost, boostEndpoint+"/eth/v1/builder/blinded_blocks", "test-cli", payload, &respPayload); err != nil {
 		log.WithError(err).Fatal("could not get payload")
 	}
 
-	if respPayload.Data == nil {
+	if respPayload.Bellatrix == nil {
```

Comment:
My suggestion does the same thing, but uses the method from the `go-builder-client` package.

```suggestion
	if respPayload.IsEmpty() {
```